i2c/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "vhost-device-i2c"
+version = "0.1.0"
+authors = ["Viresh Kumar <viresh.kumar@linaro.org>"]
+description = "vhost i2c backend device"
+repository = "https://github.com/rust-vmm/vhost-device"
+edition = "2018"
```

Comment:
Can you add all the fields that were in the root Cargo.toml?

Reply:
done